lib/sippy_cup/scenario.rb

```diff
@@ -149,13 +149,14 @@ def build(steps)
     # @param [Hash] opts A set of options to modify the message
     # @option opts [Integer] :retrans
     # @option opts [String] :headers Extra headers to place into the INVITE
+    # @option opts [String] :remote_sip_address Remote address to use in place of the SIPp constructed one
```

Comment:
So this should only impact the To header, not the actual address to which packets are sent.

We also need to do the same for the rest of the dialog, so I guess this can't actually be an option to `invite`...